Add a list of external PyGMT resources (#1210)
GenericMappingTools/pygmt: doc/external_resources.md

# External Resources

Below is a curated collection of external PyGMT resources.

> *To add your contribution to this collection, submit a recommendation via a
> [pull request](https://github.com/GenericMappingTools/pygmt/blob/master/doc/external_resources.md).*

Reviewer: I think adding to the documentation editing instructions would be helpful here and am open to your preferred wording.
```suggestion
*To add your contribution to this collection, follow [these instructions](https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#editing-the-documentation)
to submit a pull request with your recommended addition to the
[External Resources file](https://github.com/GenericMappingTools/pygmt/blob/master/doc/external_resources.md).*
```

# External Resources

Below is a curated collection of external PyGMT resources.

*To add your contribution to this collection, follow [these instructions](https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#editing-the-documentation)
to submit a pull request with your recommended addition to the
[External Resources file](https://github.com/GenericMappingTools/pygmt/blob/master/doc/external_resources.md).*

---

## Tutorials

- [2021 PyGMT course at the UAF Geophysical Institute](https://github.com/liamtoney/gi-pygmt-2021) by Liam Toney
- [Remote Online Sessions for Emerging Seismologists (ROSES): Unit 8 - PyGMT](https://www.iris.edu/hq/inclass/lesson/728) by Liam Toney

> - [PyGMT Tutorial in 2021](https://github.com/MIGG-NTU/PyGMT2021) by MIGG-NTU

Author: @core-man Are you OK with "MIGG-NTU"? Or should your name be listed here, since you're the only contributor of the https://github.com/MIGG-NTU/PyGMT2021 tutorial?

Reviewer 2: Please use MIGG-NTU.

> - [PyGMT Workshop at FOSS4G Oceania 2019](https://github.com/GenericMappingTools/foss4g2019oceania) by Wei Ji

Reviewer: Before merging, please confirm whether @weiji14 prefers this or full name format here and in L21.

Reviewer 3: Prefer to include my surname here :)

```suggestion
- [PyGMT Workshop at FOSS4G Oceania 2019](https://github.com/GenericMappingTools/foss4g2019oceania) by Wei Ji Leong
```

> ## Examples

Reviewer 3: Could we change this to be a bit more descriptive? Something like "Examples from Publications and Posters", or something along those lines?

Author: Done in 1a1670759b4a48b49558d93c5052cdcf7e326b80.

- [GMT and PyGMT plotting examples](https://github.com/michaelgrund/GMT-plotting) by Michael Grund

> - [NZ Antarctic Science Conference 2021 poster](https://github.com/weiji14/nzasc2021) by Wei Ji

Reviewer 3: ```suggestion
- [NZ Antarctic Science Conference 2021 poster](https://github.com/weiji14/nzasc2021) by Wei Ji Leong
```
